apps/bors_github/lib/github.ex

```diff
@@ -62,7 +62,7 @@ defmodule BorsNG.GitHub do
     from: bitstring,
     to: bitstring,
     commit_message: bitstring,
-    }) :: %{commit: bitstring, tree: binary} | :conflict
+    }) :: %{commit: binary, tree: binary} | :conflict
```

Comment:
It actually wasn't needed. The reason I did it is because [`binary` is a more specific typespec than `bitstring`](https://hexdocs.pm/elixir/typespecs.html#types-and-their-syntax). Namely, if you assert that the type is `binary`, you assert that the number if bits in the bitstring is a multiple of 8 (which is always true for UTF-8).